web/app/main.js

```diff
@@ -1111,6 +1111,24 @@ function dataLoaded(err, clientVersion, argCallerLocation, state, argSolar, argW
         countryTooltip.hide();
     });
 
+    // Add search bar handler
+    var selector = d3.select('.country-search-bar input')
```

Comment:
Would be nice to write ES6 code instead (we're slowly moving everything to ES6).
It's ok if you can't do it right now.